custom_components/tuya_local/devices/make_sky_blue_MPPT.yaml

```diff
@@ -0,0 +1,264 @@
+name: MPTT Make Sky Blue
+products:
+  - id: 8dgup45mwlbnguby
+    manufacturer: Make Sky Blue
+    model: v123
+entities:
+  - entity: sensor
+    class: string
+    category: diagnostic
+    name: charge_mode
+    dps:
+      - id: 101
+        type: string
+        name: charge_mode
+        mapping:
+          - dps_val: Standby
+            value: mode_1
+          - dps_val: mode_2
+            value: mode_2
+          - dps_val: mode_3
+            value: mode_3
+          - dps_val: MPPT
+            value: mode_4
+          - dps_val: mode_5
+            value: mode_5
+  - entity: sensor
+    class: voltage
+    category: diagnostic
+    name: pv_voltage
+    dps:
+      - id: 102
+        type: integer
+        name: pv_voltage
+        unit: V
+        range:
+          min: 0
+          max: 10000
+        mapping:
+          - scale: 10
+        step: 1
```

Comment:
remove this step line. 

To use a step, it needs to be indented to go inside the mapping, directly under the scale. But a step of 1 is redundant, so can just be removed. Also, step is redundant for sensors, because whatever value comes back from the device will be used. Only for entities that can be changed by the user might step be needed to limit how much they can change it by.